### DISCUSSION TITLE
Clarification on Lists reply policy

### DISCUSSION THREAD

**Discussion by TymeForChange** [mastodon/mastodon#25688] [Q&A]
I'd like some clarification on the Lists reply policy.

1. "Show replies to any followed user" -- This means any use that I follow? They need not follow back?
2. "No one" -- What is the use case for this? Why would one want to select that option?
3. "Show replies to members of the list" -- Why Reply To List (only) but no Post to List (only)? This seems counter-intuitive to me.

I made a good use case for Post To List only (ref:  [https://github.com/mastodon/mastodon/issues/20528]). If the fuction (/lookup table) already exists to reply to members of a list, why not post to the list by means of the List ID? It seems odd that one would _not_ be able to post directly to the list (a 1st step) but can reply to list (a 2nd step).

![scrList-replies-policy](https://github.com/mastodon/mastodon/assets/56969001/a4d3311a-1732-4e86-951e-a2d2994d76f4)

**Comment by trwnh** [comment]
1. yes
2. to show original posts only
3. it works off of mentions

**Comment by TymeForChange** [comment]
Hmmm. That actually just raised more questions, at least question.   Are these settings meant to filter replies _display_ on the timeline? Or, how replies are delivered to the timeline?  From your response, it seems the former. In that case,  the description word "to" really should read "from". That is a big semantics difference!

**Comment by trwnh** [comment]
no, it's "to"

say you have a list with A and B in it, and C not in it. you follow all three of these people. you also have the following posts:

```
A: post 1, top-level
A: post 2, self-reply to post 1
B: @A post 3, in reply to post 1
C: @A post 4, in reply to post 1
A: @B post 5, in reply to post 3
A: @C, post 6, in reply to post 4
```

- `followed` will show you all posts except for post 4. notably, post 6 will show up because you follow C, even though C is not in the list.
- `list` will show you all posts except for posts 4 and 6. notably, posts 3 and 5 will show up because they are replies to members of the list (A and B).
- `none` will show you only posts 1 and 2. notably, post 2 is included because self-replies are not considered the same as replies to other people.

**Comment by TymeForChange** [comment]
So, the word "of" or "from" should be used rather than "to" in the option descriptions -- docs & UI. (Colloquially, most English-speakers would probably say 'from'.)

Because your stated scenario describes this:

Show [me] replies from any followed user;
Show [me] replies from members of the list;
Show [me] replies from no one.

**Reply by JunyuKuang** [reply]
A list can only shows replies from people on that list. The list cannot "show [you] replies from any followed user."
Using "to" instead of "from" is the accurate way to describe the function.
